src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,58 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.domain.strategy.MoveStrategy;
+
+public class Cars {
+
+    List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars of(List<String> nameList) {
+        List<Car> cars = new ArrayList<>();
+        for (String name : nameList) {
+            cars.add(new RacingCar(name));
+        }
+        return new Cars(cars);
+    }
+
+    public void moveAll(MoveStrategy moveStrategy) {
+        for (Car car : cars) {
+            car.move(moveStrategy);
+        }
+    }
+
+    public List<Car> getWinners() {
+        int maxPosition = getMaxPosition();
+
+        return cars.stream()
+            .filter(car -> car.isSamePosition(maxPosition))
+            .collect(Collectors.toList());
+    }
+
+    private int getMaxPosition() {
+        return cars.stream()
+            .mapToInt(Car::getPosition)
+            .max().orElseThrow(IllegalStateException::new);
```

Comment:
아래와 같이 stream을 사용할 때는 하나의 `한 줄에 하나의 메서드`만 존재하도록 변경해주세요! 🙂
변경하게 되면, 아래의 사진처럼 인텔리제이가 각 메서드의 반환 타입을 명시해줘서 코드를 짤 때, 많은 도움을 받을 수 있어요! 🤗

되도록 적절한 `예외 메시지`를 등록해주세요 😃
```suggestion
            .max()
            .orElseThrow(() -> new IllegalStateException("예외 처리 메시지"));
```
![image](https://user-images.githubusercontent.com/47850258/151633371-d8cc57a5-d0f7-4ab9-9565-159cd656418d.png)